test_conformance/computeinfo/main.cpp

```diff
@@ -370,6 +396,62 @@ int getImageInfo(cl_device_id device, const version_t& version)
 
     return num_errors;
 }
+int getPlatformConfigInfo(cl_platform_id platform, config_info* info)
+{
+    int err = CL_SUCCESS;
+    int size_err = 0;
+    size_t config_size_set;
+    size_t config_size_ret;
+    switch (info->config_type)
+    {
+        case type_string:
+            err = clGetPlatformInfo(platform, info->opcode, 0, NULL,
+                                    &config_size_set);
+            info->config.string = NULL;
+            if (err == CL_SUCCESS && config_size_set > 0)
+            {
+                info->config.string = (char*)malloc(config_size_set);
+                err = clGetPlatformInfo(platform, info->opcode, config_size_set,
+                                        info->config.string, &config_size_ret);
+                size_err = config_size_set != config_size_ret;
+            }
+            break;
+        case type_cl_name_version_array:
+            err = clGetPlatformInfo(platform, info->opcode, 0, NULL,
+                                    &config_size_set);
+            info->config.cl_name_version_array = NULL;
+            if (err == CL_SUCCESS && config_size_set > 0)
+            {
+                info->config.cl_name_version_array = (cl_name_version*)malloc(
+                    config_size_set * sizeof(cl_name_version));
+                err = clGetPlatformInfo(platform, info->opcode, config_size_set,
+                                        info->config.cl_name_version_array,
+                                        &config_size_ret);
+                size_err = config_size_set != config_size_ret;
+                info->opcode_ret_size = config_size_ret;
+            }
+            break;
+        case type_cl_name_version:
+            err = clGetPlatformInfo(platform, info->opcode, 0, NULL,
+                                    &config_size_set);
+            if (err == CL_SUCCESS && config_size_set > 0)
+            {
+                err = clGetPlatformInfo(platform, info->opcode, config_size_set,
+                                        &info->config.cl_name_version_single,
+                                        &config_size_ret);
+            }
+            size_err = config_size_set != config_size_ret;
+            break;
+        default:
+            log_error("Unknown config type: %d\n", info->config_type);
+            break;
+    }
+    if (err || size_err)
+        log_error("\tFailed clGetPlatformInfo for %s.\n", info->opcode_name);
+    if (err) print_error(err, "\t\clGetPlatformInfo failed.");
```

Comment:
I was able to re-trigger the Travis build and this line is generating an error:

```
/Users/travis/build/KhronosGroup/OpenCL-CTS/test_conformance/computeinfo/main.cpp:451:34: error: unknown escape sequence '\c' [-Werror,-Wunknown-escape-sequence]
    if (err) print_error(err, "\t\clGetPlatformInfo failed.");
```

Fix is straightforward:

```suggestion
    if (err) print_error(err, "\t\tclGetPlatformInfo failed.");
```